magento/inventory
# MSI-666: When we apply "Save & Duplicate" for existing Simple product - stock status for new product should be "Out of stock".

### Description


### Fixed Issues (if relevant)

1. magento-engcom/msi#666: When we apply "Save & Duplicate" for existing Simple product - stock status for new product should be "Out of stock".

### Manual testing scenarios

1. Magento 2 in Single Source Mode.
2. Create a Simple Product, make it 'In Stock'.
3. Duplicate Simple Product.
4. A newly created Product should be 'Out Of Stock'

### Contribution checklist
